2021-12-02 19:00:25,040 [cuckoo.core.scheduler] INFO: Task #19: acquired machine cuckoo1 (label=cuckoo1)
2021-12-02 19:00:25,040 [cuckoo.core.resultserver] DEBUG: Now tracking machine 192.168.122.254 for task #19
2021-12-02 19:00:25,041 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Replay
2021-12-02 19:00:25,047 [cuckoo.auxiliary.sniffer] INFO: Started sniffer with PID 25510 (interface=qemubr, host=192.168.122.254)
2021-12-02 19:00:25,047 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Sniffer
2021-12-02 19:00:25,071 [cuckoo.machinery.qemu] DEBUG: Starting vm cuckoo1
2021-12-02 19:00:25,092 [cuckoo.machinery.qemu] DEBUG: Executing QEMU ['qemu-system-x86_64', '-display', 'none', '-m', '1024M', '-hda', '/home/crochetch/Documents/PhD/VMImages/images/snapshot_cuckoo1.qcow2', '-net', 'tap,ifname=tap_cuckoo1,script=no,downscript=no', '-net', 'nic,macaddr=52:54:00:12:34:56', '-enable-kvm']
2021-12-02 19:00:25,122 [cuckoo.core.guest] INFO: Starting analysis #19 on guest (id=cuckoo1, ip=192.168.122.254)
2021-12-02 19:00:25,161 [cuckoo.core.guest] INFO: Guest is running Cuckoo Agent 0.10 (id=cuckoo1, ip=192.168.122.254)
2021-12-02 19:00:30,470 [cuckoo.core.guest] DEBUG: Uploading analyzer to guest (id=cuckoo1, ip=192.168.122.254, monitor=latest, size=1720485)
2021-12-02 19:00:30,819 [cuckoo.core.resultserver] DEBUG: Task #19: live log analysis.log initialized.
2021-12-02 19:00:31,835 [cuckoo.core.guest] WARNING: cuckoo1: analysis #19 caught an exception
Traceback (most recent call last):
  File "C:/tmpisdltw/analyzer.py", line 808, in <module>
    success = analyzer.run()
  File "C:/tmpisdltw/analyzer.py", line 657, in run
    pids = self.package.start(self.target)
  File "C:\tmpisdltw\modules\packages\exe.py", line 20, in start
    os.rename(path, new_path)
WindowsError: [Error 183] Cannot create a file when that file already exists

2021-12-02 19:00:31,843 [cuckoo.core.plugins] DEBUG: Stopped auxiliary module: Replay
2021-12-02 19:00:31,843 [cuckoo.core.plugins] ERROR: Unable to stop auxiliary module: Sniffer
Traceback (most recent call last):
  File "/home/crochetch/Documents/toolchain_malware_analysis/penv-2.7/lib/python2.7/site-packages/Cuckoo-2.0.7-py2.7.egg/cuckoo/core/plugins.py", line 164, in stop
    module.stop()
  File "/home/crochetch/Documents/toolchain_malware_analysis/penv-2.7/lib/python2.7/site-packages/Cuckoo-2.0.7-py2.7.egg/cuckoo/auxiliary/sniffer.py", line 156, in stop
    (out, err, faq("permission-denied-for-tcpdump"))
CuckooOperationalError: Error running tcpdump to sniff the network traffic during the analysis; stdout = '' and stderr = 'tcpdump: qemubr: No such device exists\n(SIOCGIFHWADDR: No such device)\n'. Did you enable the extra capabilities to allow running tcpdump as non-root user and disable AppArmor properly (the latter only applies to Ubuntu-based distributions with AppArmor, see also https://cuckoo.sh/docs/faq/index.html#permission-denied-for-tcpdump)?
2021-12-02 19:00:31,848 [cuckoo.machinery.qemu] DEBUG: Stopping vm cuckoo1
2021-12-02 19:00:31,859 [cuckoo.core.resultserver] DEBUG: Stopped tracking machine 192.168.122.254 for task #19
2021-12-02 19:00:31,859 [cuckoo.core.resultserver] DEBUG: Cancel <Context for LOG> for task 19
2021-12-02 19:00:31,869 [cuckoo.core.scheduler] DEBUG: Released database task #19
2021-12-02 19:00:31,886 [cuckoo.core.plugins] DEBUG: Executed processing module "AnalysisInfo" for task #19
2021-12-02 19:00:31,886 [cuckoo.processing.behavior] WARNING: Analysis results folder does not contain any behavior log files.
2021-12-02 19:00:31,886 [cuckoo.core.plugins] DEBUG: Executed processing module "BehaviorAnalysis" for task #19
2021-12-02 19:00:31,887 [cuckoo.core.plugins] DEBUG: Executed processing module "Dropped" for task #19
2021-12-02 19:00:31,887 [cuckoo.core.plugins] DEBUG: Executed processing module "DroppedBuffer" for task #19
2021-12-02 19:00:31,888 [cuckoo.core.plugins] DEBUG: Executed processing module "MetaInfo" for task #19
2021-12-02 19:00:31,888 [cuckoo.core.plugins] DEBUG: Executed processing module "ProcessMemory" for task #19
2021-12-02 19:00:31,888 [cuckoo.core.plugins] DEBUG: Executed processing module "Procmon" for task #19
2021-12-02 19:00:31,889 [cuckoo.core.plugins] DEBUG: Executed processing module "Screenshots" for task #19
2021-12-02 19:00:32,094 [cuckoo.core.plugins] DEBUG: Executed processing module "Static" for task #19
2021-12-02 19:00:32,110 [cuckoo.core.plugins] DEBUG: Executed processing module "Strings" for task #19
2021-12-02 19:00:32,121 [cuckoo.core.plugins] DEBUG: Executed processing module "TargetInfo" for task #19
2021-12-02 19:00:32,122 [cuckoo.processing.network] WARNING: The PCAP file does not exist at path "/home/crochetch/Documents/toolchain_malware_analysis/src/sandboxes/.cuckoo/storage/analyses/19/dump.pcap".
2021-12-02 19:00:32,122 [cuckoo.core.plugins] DEBUG: Executed processing module "NetworkAnalysis" for task #19
2021-12-02 19:00:32,122 [cuckoo.core.plugins] DEBUG: Executed processing module "Extracted" for task #19
2021-12-02 19:00:32,122 [cuckoo.core.plugins] DEBUG: Executed processing module "TLSMasterSecrets" for task #19
2021-12-02 19:00:32,124 [cuckoo.core.plugins] DEBUG: Executed processing module "Debug" for task #19
2021-12-02 19:00:32,124 [cuckoo.core.plugins] DEBUG: Running 0 signatures
2021-12-02 19:00:32,128 [cuckoo.core.plugins] DEBUG: Executed reporting module "JsonDump"
2021-12-02 19:00:32,164 [cuckoo.core.plugins] DEBUG: Executed reporting module "SingleFile"
2021-12-02 19:00:32,171 [cuckoo.core.plugins] DEBUG: Executed reporting module "MongoDB"
2021-12-02 19:00:32,171 [cuckoo.core.scheduler] INFO: Task #19: reports generation completed
2021-12-02 19:00:32,176 [cuckoo.core.scheduler] INFO: Task #19: analysis procedure completed
